src/ReactiveUI.Events.XamForms/ReactiveUI.Events.XamForms.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.3</TargetFrameworks>
+    <PackageTargetFallback Condition="'$(TargetFramework)' == 'netstandard1.3'">
```

Comment:
I think worth dropping netstandard 1.3 from the ReactiveUI.Events one to complement this.